src/components/manage/Widgets/eeaCountries.js

```diff
@@ -0,0 +1,41 @@
+export const eeaCountries = [
```

Comment:
We need a way to let the user to easily select group of countries:
```
EEA32
EEA33
EEA39
EFTA4
EU15
EU25
EU27
EU28
Pan-Europe
```

Thus, when one select `EEA32`, all countries within this group will be added to the selection.

One way of approaching this, is to add a new key to this vocabulary, called `groups` as one country can be part of more than one of the above groups.

Reply:
Thinking of making a new Select Widget for selecting the Country group and according to that the countries in the list gets reshuffled each time the user selects new group. Same as what happens now in eea website.

Reply:
<img width="367" alt="Screenshot 2020-10-08 at 5 19 57 PM" src="https://user-images.githubusercontent.com/22280901/95454749-7fd90d00-098a-11eb-9d89-d2b68702a57a.png">
Something like this.